feature-libs/quote/components/quote-actions-by-role/quote-actions-by-role.component.spec.ts

```diff
@@ -203,6 +211,88 @@ describe('QuoteActionsByRoleComponent', () => {
     );
   });
 
+  describe('Threshold check', () => {
+    it('should let submit button enabled is threshold is met', () => {
+      const allowedActions = [
+        { type: QuoteActionType.SUBMIT, isPrimary: true },
+      ];
+      const newMockQuoteWithSubmitAction: QueryState<Quote> = {
+        error: false,
+        loading: false,
+        data: {
+          ...mockQuote,
+          allowedActions: allowedActions,
+        },
```

Comment:
this 'newMockQuoteWithSubmitAction' (or similar) is declared in several tests. I would suggest to move it to the 'before' callback with default state and just modify the parts relevant for the test within the test. Such as setting the total value or actions.
declare it like below, so you do not have to deal with data possibly being undefined strict mode checks
```
let newMockQuoteWithAction: {
    loading: boolean;
    error: false | Error;
    data: Quote;
  };
```
and then in before
```
 newMockQuoteWithSubmitAction = {
      error: false,
      loading: false,
      data: {
        ...mockQuote,
      },
    };
```
so in the test it can be written as
```
      newMockQuoteWithSubmitAction.data.allowedActions = [
        { type: QuoteActionType.SUBMIT, isPrimary: true },
      ];
```